sysutils/pfSense-pkg-apcupsd/Makefile

```diff
@@ -35,6 +35,10 @@ do-install:
 		${STAGEDIR}${PREFIX}/pkg
 	${INSTALL_DATA} ${FILESDIR}${PREFIX}/www/apcupsd_status.php \
 		${STAGEDIR}${PREFIX}/www
+	${INSTALL_DATA} ${FILESDIR}${PREFIX}/www/widgets/widgets/apcupsd.widget.php \
+		${STAGEDIR}${PREFIX}/www/widgets/windgets
```

Comment:
`${STAGEDIR}${PREFIX}/www/widgets/widgets` must be created earlier as other directories above.  Also note there is a extraneous n at  `wiNdgets`

Reply:
Changes submitted